.github/workflows/publish_container.yml

```diff
@@ -1,16 +1,16 @@
-name: Publish version tag
-on:
-  merge_group:
-  pull_request:
-    branches:
-      - main
-    paths-ignore:
-      - ".github/**"
-      - ".gitignore"
-      - "**/*.md"
-      - "charts/**"
-      - "Makefile"
-    types: [ closed ]
+#name: Publish version tag
+#on:
+#  merge_group:
+#  pull_request:
+#    branches:
+#      - release_v1
+#    paths-ignore:
+#      - ".github/**"
+#      - ".gitignore"
+#      - "**/*.md"
+#      - "charts/**"
+#      - "Makefile"
+#    types: [ closed ]
```

Comment:
Shouldn't we leave this on `main` and add the `release_v1` branch when it is time to start creating artifacts from this branch?